### PR TITLE
Ana/219 send and receive amounts not showing

### LINKED ISSUE #219: Transaction amounts not shown
Apologies if this is well known - just adding these as I notice it.

Amount it not shown for SendTx but seems with work for withdraw rewards:

image

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #219

All good

image

### CODE REVIEW

File: apis/cosmos-reducers.js [hunk 1]
```diff
@@ -584,10 +584,7 @@ export function transactionReducer(transaction) {
         return coinReducer(fee, coinLookup)
       })
     }
-    const {
-      claimMessages,
-      otherMessages,
-    } = transaction.tx.body.messages.reduce(
+    const { claimMessages, otherMessages } = transaction.tx.value.msg.reduce(
```
Comment (faboweb) [inline apis/cosmos-reducers.js:587]: no

File: apis/cosmos-reducers.js [hunk 2]
```diff
@@ -622,7 +619,7 @@ export function transactionReducer(transaction) {
           transaction
         ),
         timestamp: transaction.timestamp,
-        memo: transaction.tx.body.memo,
+        memo: transaction.tx.value.memo,
```
Comment (faboweb) [inline apis/cosmos-reducers.js:622]: no

File: apis/cosmos-reducers.js (hunk 1 above)
Comment (faboweb) [reply apis/cosmos-reducers.js:587]:
```suggestion
    const { claimMessages, otherMessages } = transaction.tx.body.messages.reduce(
```

File: apis/cosmos-reducers.js (hunk 2 above)
Comment (faboweb) [reply apis/cosmos-reducers.js:622]:
```suggestion
        memo: transaction.tx.body.memo,
```

File: apis/cosmos-reducers.js (hunk 1 above)
Comment (Bitcoinera) [reply apis/cosmos-reducers.js]: I tested with both e-Money and Cosmos Hub :shrug:

File: apis/cosmos-reducers.js (hunk 2 above)
Comment (Bitcoinera) [reply apis/cosmos-reducers.js]: I tested with both e-Money and Cosmos Hub :shrug:

File: apis/cosmos-reducers.js [hunk 3]
```diff
@@ -619,7 +619,7 @@ export function transactionReducer(transaction) {
           transaction
         ),
         timestamp: transaction.timestamp,
-        memo: transaction.tx.value.memo,
+        memo: transaction.tx.body.memo,
```
Comment (Bitcoinera) [inline apis/cosmos-reducers.js]: Now it will probably fail? :eyes:

Comment (faboweb) [reply apis/cosmos-reducers.js]: the reducers are for stargate... the cosmoshub-3 reducers are cosmos-reducers-v39

**Pull request merged**